crates/cubecl-wgpu/src/compiler/wgsl/compiler.rs

```diff
@@ -26,15 +26,21 @@ pub struct WgslCompiler {
 }
 
 impl core::fmt::Debug for WgslCompiler {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+    fn fmt(
+        &self,
+        f: &mut std::fmt::Formatter<'_>,
+    ) -> std::fmt::Result {
```

Comment:
Is there a reason why the fommatting is different? What Rust version / rust-fmt are you using?